tests/foreman/api/test_errata.py

```diff
@@ -55,6 +54,8 @@
 
 CUSTOM_REPO_URL = FAKE_6_YUM_REPO
 CUSTOM_REPO_ERRATA_ID = FAKE_2_ERRATA_ID
+DISTRO_RHEL6 = settings.distro.image_el6
```

Comment:
robottelo/test.py:130 

TestCase.setUpClass calls settings.configure()  you have to put it in inherited setUpClass
You need class instance for settings to work.